README.md

```diff
@@ -18,6 +18,7 @@
 
 [![Build](https://github.com/Belco90/octoclairvoyant/actions/workflows/build.yml/badge.svg)](https://github.com/Belco90/octoclairvoyant/actions/workflows/build.yml)
```

Comment:
```suggestion
```

I think this badge will break once we merge the PR.

You don't care about `yarn build ` results via GitHub Actions then? I think Vercel is already building the page, so we can use the Vercel build status instead. 😄

Reply:
Oh well, the workflow has to point to `e2e.yml` instead.

> You don't care about yarn build results via GitHub Actions then? I think Vercel is already building the page, so we can use the Vercel build status instead. 😄

I don't follow you here, sorry! This is just adding a badge about the E2E results status, not the build.

Reply:
```suggestion
[![e2e](https://github.com/Belco90/octoclairvoyant/actions/workflows/e2e.yml/badge.svg)](https://github.com/Belco90/octoclairvoyant/actions/workflows/e2e.yml)
```

Oh, you're replacing `build.yml` with a new `e2e.yml` workflow.
In that case we should replace the `build` badge with a new `e2e` badge. 😉

Reply:
Aah sorry, you were right! The build pipeline doesn't exist anymore, so I need to remove that badge. My bad!